spring-context/src/main/java/org/springframework/context/support/AbstractApplicationContext.java

```diff
@@ -713,9 +713,8 @@ protected void invokeBeanFactoryPostProcessors(ConfigurableListableBeanFactory b
 	}
 
 	/**
-	 * Instantiate and invoke all registered BeanPostProcessor beans,
+	 * Instantiate and register all BeanPostProcessor beans,
 	 * respecting explicit order if given.
-	 * <p>Must be called before any instantiation of application beans.
```

Comment:
Please reinstate this line.

Reply:
Do you mean to make a new commit to reinstate the line?

Reply:
The line is already reinstated.